util/MsSqlMigratorUtility/Program.cs

```diff
@@ -15,60 +12,26 @@ private static int Main(string[] args)
     public void Execute(
         [Operand(Description = "Database connection string")]
         string databaseConnectionString,
-        [Option('v', "verbose", Description = "Enable verbose output of migrator logs")]
```

Comment:
❓ Nothing emitted debug messages so this seemed unnecessary. Am I missing something here? Do we use this anywhere, like with automation?

Reply:
It's used in the automations. If the `verbose` flag is set during execution, the Migrator logs the SQL scripts applied during the run. If we want to remove the `verbose` option, I would like to set the `enableLogging` param in the `MigrateMsSqlDatabaseWithRetries` method to always be `true`, like you did in lines 28 and 32. This is helpful to see which scripts are run in the automation (to confirm if all needed have been applied).

Reply:
OK, so even though you invoked this as "verbose", what it was actually doing is simply logging. It configured a debug-level log emission, but nothing logs at that level. I think this is fine then and at some point the automations could omit that parameter once this merges.

You can see in the Action run logs for this PR that you still get those migration messages. Please review the database testing run if you don't mind to ensure this works as you (DevOps) expect.

Reply:
I just checked it, and it works as expected 😃

Reply:
@withinfocus We need to ensure that this change is backwards compatible. If we use the `-v` flag after removing the verbose option, will the `MsSqlMigratoeUtility` run as expected, or will it throw an error stating that it does not know the `-v` option?

Reply:
Sigh, I hacked around on this and can't get `Unrecognized option '-v'` to leave. I pushed https://github.com/bitwarden/server/pull/3910/commits/32b0801a6ebfbb8b54dc901126822af3a277d966 just now for this -- is that all that's needed?

Reply:
The `${@}` should stay, as it passes other args, e.g. the `--repeatable` or `--folder`.